data-prepper-plugins/opensearch/src/test/java/com/amazon/dataprepper/plugins/sink/opensearch/OpenSearchIntegrationHelper.java

```diff
@@ -0,0 +1,254 @@
+/*
```

Comment:
Why is the long format license needed here?

Reply:
I copied this code from [OpenSearchRestTestCase.java](https://github.com/opensearch-project/OpenSearch/blob/main/test/framework/src/main/java/org/opensearch/test/rest/OpenSearchRestTestCase.java). As copied code it must have the long-form header and the headers from the original file.

Reply:
"I copied this code" hits me like nails on a chalkboard. :) Would other projects have this same issue and benefit from a reliable test client that doesn't bring in the OpenSearch core? I am wondering if we can standardize this for the community and make something usable rather than copying code.